app-backend/db/src/main/scala/MVTLayerDao.scala

```diff
@@ -13,6 +20,24 @@ import java.util.UUID
   * Or fieldnames? Or table name?
   */
 object MVTLayerDao {
+
+  final case class LabelTileGeometry(
+      geom: Projected[Geometry],
+      envelope: ProjectedExtent,
+      taskId: UUID,
+      labelClassId: UUID,
+      className: String,
+      colorHexCode: String
+  ) {
+    def vtValues: Map[String, Value] =
+      Map(
+        "annotation_task_id" -> VString(taskId.toString),
+        "label_class_id" -> VString(labelClassId.toString),
+        "name" -> VString(className),
+        "color_hex_code" -> VString(colorHexCode)
```

Comment:
these names match what's present in the GW UI codebase

Reply:
Yeah the `VFoo` types are an ADT to represent the different kinds of things we might want to encode in vector tile layers. It's like the `JString`, `JInt`, etc. case classes [in `circe`](https://github.com/circe/circe/blob/master/modules/core/shared/src/main/scala/io/circe/Json.scala#L369)